doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1,9 +1,33 @@
 FORMAT: 1A
-HOST: http://orion.lab.fiware.org/
+HOST: http://telefonicaid.github.io/fiware-orion/api/v2/
+TITLE: FIWARE-NGSI v2 Specification
+DATE: 15 September 2015
+VERSION: <placeholder>
```

Comment:
nice to have would be to automatically add here the commit hash using a hook ...